include/data_management/data/arrow_numeric_table.h

```diff
@@ -163,9 +164,13 @@ class DAAL_EXPORT ArrowImmutableNumericTable : public BaseArrowImmutableNumericT
         {
             const NumericTableFeature & f = (*_ddict)[i];
 
+#if ARROW_VERSION >= 15000
```

Comment:
I see some copy-paste here. Could you please introduce function to encapsulate differences between Arrow versions?

Reply:
Such a function was introduced. Thank you for comment!